rpc/websockets.go

```diff
@@ -27,27 +27,28 @@ import (
 	"sync"
 
 	"github.com/cosmos/cosmos-sdk/client"
-	"github.com/gorilla/mux"
-	"github.com/gorilla/websocket"
-	"github.com/pkg/errors"
-
 	"github.com/ethereum/go-ethereum/common"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/eth/filters"
 	"github.com/ethereum/go-ethereum/params"
 	"github.com/ethereum/go-ethereum/rpc"
-
+	evmtypes "github.com/evmos/ethermint/x/evm/types"
+	"github.com/gorilla/mux"
+	"github.com/gorilla/websocket"
+	"github.com/pkg/errors"
 	"github.com/tendermint/tendermint/libs/log"
 	rpcclient "github.com/tendermint/tendermint/rpc/jsonrpc/client"
 	tmtypes "github.com/tendermint/tendermint/types"
-
-	evmtypes "github.com/evmos/ethermint/x/evm/types"
 	"github.com/zeta-chain/zetacore/rpc/ethereum/pubsub"
 	rpcfilters "github.com/zeta-chain/zetacore/rpc/namespaces/ethereum/eth/filters"
 	"github.com/zeta-chain/zetacore/rpc/types"
 	"github.com/zeta-chain/zetacore/server/config"
 )
 
+const (
+	messageSizeLimit = 32 * 1024 * 1024 // 15MB
```

Comment:
`// 32MB`

Reply:
Just FYI, you can leave code modification suggestion directly on Github 

```suggestion
	messageSizeLimit = 32 * 1024 * 1024 // 32MB
```